packages/teams-js/src/public/teamsAPIs.ts

```diff
@@ -53,9 +53,6 @@ export namespace teamsCore {
    *
    * @param handler - The handler to invoke when the page is loaded.
    *
-   * @deprecated
-   * As of 2.14.1, please use {@link app.lifecycle.registerOnResumeHandler} instead.
-   *
```

Comment:
Is this no longer deprecated? I understood from the PR description that you want to maintain the same messages across the TJS->Host boundary (for compatibility and confusion reasons), but presumably we would still prefer developers not add new uses of the functions in this file? Or is that not true?